corehq/apps/accounting/models.py

```diff
@@ -1245,11 +1245,6 @@ def change_plan(self, new_plan_version, date_end=None,
         self.is_active = False
         self.save()
 
-        if 'date_delay_invoicing' in kwargs:
-            date_delay_invoicing = kwargs.pop('date_delay_invoicing')
-        else:
-            date_delay_invoicing = self.date_delay_invoicing
```

Comment:
it looks like the behavior is slightly different now with `date_delay_invoicing` being `None` when it's not passed in whereas before it was defaulting to `self.date_delay_invoicing`. is that intentional?

Reply:
Yes - in the uses of ```Subscription.change_plan``` where it is not passed in, it's supposed to be set to ```None```.